tests/unit/test_base_command.py

```diff
@@ -46,7 +65,7 @@ def test_log_command_success(self, tmpdir):
         log_path = tmpdir.join('log')
         cmd.main(['fake', '--log', log_path])
         with open(log_path) as f:
-            assert 'fake' == f.read().strip()[:4]
+            assert f.read().startswith('2019-01-17T06:00:37 fake')
```

Comment:
Can this be an equality if `rstrip()` is called (to remove the platform-dependent line ending I assume)?